packages/dd-trace/src/config.js

```diff
@@ -1114,6 +1123,13 @@ class Config {
     this._setTags(opts, 'tags', tags)
     this._setBoolean(opts, 'traceId128BitGenerationEnabled', options.traceId128BitGenerationEnabled)
     this._setBoolean(opts, 'traceId128BitLoggingEnabled', options.traceId128BitLoggingEnabled)
+    if (options.tracePropagationBehaviorExtract) {
```

Comment:
Do we want to support setting this programmatically? Generally I think we should move towards only allowing configs to be set through environment variables/stable config